src/server/components/script.tsx

```diff
@@ -33,7 +33,7 @@ export const Script = (options: Options): any => {
             <script
               type='module'
               async={!!options.async}
-              src={`/${scriptInManifest.file}`}
+              src={`${import.meta.env.BASE_URL}${scriptInManifest.file}`}
```

Comment:
Should we support without a trailing slash pattern?

```ts
defineConfig({
  base: '/subdir' // does not have a trailing slash
})
```

Reply:
Oh, that should be ensured